addons/web_editor/views/snippets.xml

```diff
@@ -46,6 +46,9 @@
 
     <div data-selector=".s_hr"
          data-drop-near="p, h1, h2, h3, blockquote, .s_hr"/>
+
+    <div data-js="VersionControl"
+         data-selector="[data-snippet]"/>
```

Comment:
I don't think this rule is enough to add the ability to delete those elements if required. Could you check again? If you add a data-snippet random in your DOM, you should have the alert since the system does not know the snippet but you should also be able to remove it.

Especially important for our carousel update which occured in 13.1. In the current state, people who used the carousel in 13.0 will not be able to select it again and thus delete it when they migrate to 13.2. Could you check it works with your branch?

Reply:
It is the case you can remove a random element if it has data-snippet in the DOM.

For carousel you can not select it but you can select it's items so even without `data-snippet` you would be able to select and delete it. Does not matter as it will work with `data-snippet` as stated above.
One thing with the carousel is that it has not been versioned in 13.1with `data-vxml="001"` as it did not exist yet. Should I add this in this PR in 13.2 ?

Reply:
> Does not matter as it will work with data-snippet as stated above.

Yes, that was the point, making sure data-snippet removal worked and that old carousel will be flagged with data-snippet by migration script. So yes strange, I though there was a data-vcss in 13.2 but apparently not. So yes, add data-vxml :+1: (and make sure the migration script flag those).

Reply:
Would be nice if migration from 13.0 to 13.1 was working. I was not able to do a full migration from like 13.0 or 12.0 to 13.2... Maybe it is fixed I will check that later on.

Reply:
Mmh, NSE did not explain me that it could not work :o I'll test tomorrow